RadixWallet/Clients/SecurityCenterClient/SecurityCenterClient+Live.swift

```diff
@@ -61,11 +61,13 @@ extension SecurityCenterClient {
 				return combineLatest(profiles, cloudBackups, manualBackups).map { profile, cloudBackup, manualBackup in
 					let isCloudProfileSyncEnabled = profile.appPreferences.security.isCloudProfileSyncEnabled
 
-					func hasProblem3() async -> (accounts: [AccountAddress], personas: [IdentityAddress])? {
-						guard let result = try? await deviceFactorSourceClient.unrecoverableEntities(),
-						      result.accounts.count + result.personas.count > 0
-						else { return nil }
-						return result
+					func hasProblem3or9() async -> (problem3: ProblematicAddresses?, problem9: ProblematicAddresses?) {
+						guard let result = try? await deviceFactorSourceClient.problematicEntities() else {
```

Comment:
You could also do `let problematic = try? await deviceFactorSourceClient.problematicEntities()` outside the functions, and then still have separate and simple `hasProblem3` and `hasProblem9`. Something like 
```
guard let problematic, !problematic.mnemonicMissing.isEmpty else { return nil }
return problematic.mnemonicMissing
```